### PR TITLE
Fix crud param names

### PR DESCRIPTION
**Pull request by COV-GIS**
Per https://github.com/Esri/arcgis-rest-js/issues/391

### CODE REVIEW

File: packages/arcgis-rest-feature-service/src/add.ts [hunk 1]
```diff
@@ -26,7 +27,13 @@ export interface IAddFeaturesRequestOptions
...
    * Array of JSON features to add.
    */
-  adds: IFeature[];
+  features: IFeature[];
+  /**
+   * Deprecated. Please use `features` instead.
+   *
+   * Array of JSON features to add.
+   */
+  adds?: IFeature[];
```
Comment (jgravois) [inline packages/arcgis-rest-feature-service/src/add.ts]:
```js
/**
* Deprecated. Please use `features` instead.
```
~~Array of JSON features to add.~~

these two will show up right next to each other in the API reference, so no need to repeat the meat of the description.

**Comment by COV-GIS** [comment]
Hey @jgravois. I got a new work station. Build and test work just fine but I'm having issues running `docs:serve` to check the docs. All the tasks run w/o error. `typedoc.json` is created. The assets are all copied. But no html files are being generated. Any thoughts on why?

**Comment by jgravois** [comment]
> no html files are being generated. Any thoughts on why?

yup. acetate doesn't write output files to disk when you're running the dev server. they're only in memory ✨.

its a bit of a gotcha, but until https://github.com/patrickarlt/acetate/pull/74 lands, we all have to tack `/arcgis-rest-js/` onto the url thats spawned manually.

[`http://localhost:3000/arcgis-rest-js/`](http://localhost:3000/arcgis-rest-js/)

**Comment by COV-GIS** [comment]
It must have just opened `http://localhost:3000/arcgis-rest-js/` on my old machine. Thanks.

**Comment by jgravois** [comment]
> It must have just opened http://localhost:3000/arcgis-rest-js/ on my old machine. Thanks.

nah, its a regression i introduced in #341.

if you could make the [breadcrumb](https://github.com/Esri/arcgis-rest-js/blob/master/CONTRIBUTING.md#building-the-documentation-site-locally) i already dropped more explicit, it would be _sincerely_ appreciated.

**Comment by COV-GIS** [comment]
That looks good to me @jgravois. Had I seen that I would have known.

**Pull request merged**